<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlDockWidget</class>
 <widget class="QDockWidget" name="CameraControlDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>567</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Camera Control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-7</y>
         <width>269</width>
         <height>550</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="CameraSetting_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Camera setting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="CameraYFixed_radioButton">
             <property name="text">
              <string>Camera y fixed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="FollowWario_radioButton">
             <property name="text">
              <string>Follow Wario</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="VerticalSeperate_radioButton">
             <property name="text">
              <string>Vertical Seperate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="UseCameraLimitators_radioButton">
             <property name="text">
              <string>Use Camera boxes</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ExistingLimitators_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Camera boxes settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" rowspan="2">
              <widget class="QListView" name="CameraLimitators_listView"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="AddCameraLimitator_pushButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="DeleteCameraLimitator_pushButton">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_LimitatorSetting">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_LimitatorTypes">
               <property name="text">
                <string>Camera box type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CameraLimitatorTypePicker_comboBox"/>
             </item>
             <item>
              <widget class="QLabel" name="label_Position_Size">
               <property name="text">
                <string>Position &amp; Size (Hex)</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox_x1">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_y">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_x">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QSpinBox" name="spinBox_y1">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_width">
                 <property name="text">
                  <string>Width:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBox_width">
                 <property name="minimum">
                  <number>15</number>
                 </property>
                 <property name="maximum">
                  <number>251</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_height">
                 <property name="text">
                  <string>Height</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QSpinBox" name="spinBox_height">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>251</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_sideoffset">
               <property name="text">
                <string>Side offset of caemra box (Hex)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="LimitatorSideOffset_spinBox">
               <property name="minimum">
                <number>-255</number>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PositionofTriggerBlock">
               <property name="text">
                <string>Position of the trigger block (Hex)</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QSpinBox" name="TriggerBlockPositionX_spinBox">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="TriggerBlockPositionY_spinBox">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>16</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
